From 81785c1b9a9c1b797e7406a3542c01d972c61315 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sun, 7 Aug 2022 16:45:30 +0200
Subject: [PATCH] block: Do not include linux/fs.h any more

glibc 2.36 changed the definition of enum fsconfig_command in
sys/mount.h. This definition collides with the same definition from
linux/fs.h now. Remove the include of linux/fs.h. This compiles still
with musl too.

musl and glibc provide the defines linux/fs.h was included for in
sys/mount.h since some years.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 block.c | 2 --
 1 file changed, 2 deletions(-)

--- a/block.c
+++ b/block.c
@@ -33,8 +33,6 @@
 #include <sys/wait.h>
 #include <sys/sysmacros.h>
 
-#include <linux/fs.h>
-
 #include <uci.h>
 #include <uci_blob.h>
 
